smartPly.py

```diff
@@ -1,4 +1,5 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
What seems to be a copy of this string has already been refactored in this PR https://github.com/malus-security/sandscout/pull/1. Is it worth keeping both of them?